pkg/releasesync/releasesync_test.go

```diff
@@ -166,3 +170,119 @@ func buildReleaseDetails(hasFailedBlockingJobs bool) ReleaseDetails {
 	return releaseDetails
 
 }
+
+func TestChangeLog(t *testing.T) {
+
+	data, err := ioutil.ReadFile(`OCPCRT-74-pr-test.json`)
+	if err != nil {
+		t.Fatal("Failed to read test file")
+	}
+
+	releaseDetails := ReleaseDetails{}
+
+	if err := json.Unmarshal(data, &releaseDetails); err != nil {
+		panic(err)
+	}
+
+	if len(releaseDetails.ChangeLog) == 0 {
+		t.Fatal("Failed unmarshalling")
+	}
+
+	changeLogStr := string(releaseDetails.ChangeLog)
+	releaseChangeLog := models.ReleaseTag{}
+	changelog := NewChangelog("amd64", changeLogStr)
+	releaseChangeLog.KubernetesVersion = changelog.KubernetesVersion()
+	releaseChangeLog.CurrentOSURL, releaseChangeLog.CurrentOSVersion, releaseChangeLog.PreviousOSURL, releaseChangeLog.PreviousOSVersion, releaseChangeLog.OSDiffURL = changelog.CoreOSVersion()
+	releaseChangeLog.PreviousReleaseTag = changelog.PreviousReleaseTag()
+	releaseChangeLog.Repositories = changelog.Repositories()
+	releaseChangeLog.PullRequests = changelog.PullRequests()
+
+	releaseChangeLogJSON := parseChangeLogJSON("test", releaseDetails.ChangeLogJSON)
+
+	if releaseChangeLogJSON.KubernetesVersion != releaseChangeLog.KubernetesVersion {
+		t.Fatalf("ReleaseChangeLog Kubernetes versions don't match.  ChangeLog: %s, ChangeLogJson: %s", releaseChangeLog.KubernetesVersion, releaseChangeLogJSON.KubernetesVersion)
+	}
```

Comment:
Nit, but all these if's are challenging to read for me. If you can't directly use `reflect.DeepEqual` to just compare both structs entirely, I think this would look nicer to use helpers from testify (it's already vendored in sippy):

```go
assert.EqualValues(t, releaseChangeLogJSON.KubernetesVersion, releaseChangeLog.KubernetesVersion, "Kubernetes version")
```

It will produce a diff and a helpful error message and it's a bit more concise.

Reply:
Honestly I wasn't sure if we wanted the test going forward.  I used it get familiar with / build out the changes.  I don't mind putting some time into refactoring and cleaning it up.  I also didn't know if once the JSON was available we would remove the HTML parsing and not even need the test going forward.

Reply:
Yea I guess that makes sense, I'm fine if you just want to `/hold cancel` and let this merge as is.  We can remove the HTML parsing code and the test once the new json thing is ready